src/Entities/Client.php

```diff
@@ -64,21 +64,21 @@ public function __construct(
 
```

Comment:
Should it be `return $this->userAgent->get()`?

Reply:
yes, but how it now?